packages/playground/src/index.css

```
@@ -1,33 +1,27 @@
   :root {
     --top-bar-height: 40px;
     --color-thumb: var(--surface4);
     --color-thumb-hover: var(--surface6);
     --color-track: transparent;
+
+    /*
+     * Override playground-ui semantic font tokens with locally-bundled fonts.
+     * Branded Greed (Mastra Cloud) only ships in the platform repo; here we
+     * keep TasaExplorer as the Studio display font to match prior releases.
+     */
+    --font-display: TasaExplorer, ui-serif, georgia, cambria, 'Times New Roman', times, serif;
```

Stylelint (17.10.0)

[error] Expected "TasaExplorer" to be "tasaexplorer" (value-keyword-case)

(value-keyword-case)